models/default_local_provider.go

```diff
@@ -18,39 +18,59 @@ import (
 // DefaultLocalProvider - represents a local provider
 type DefaultLocalProvider struct {
 	*MapPreferencePersister
+	ProviderProperties
 	SaaSBaseURL           string
 	ResultPersister       *BitCaskResultsPersister
 	SmiResultPersister    *BitCaskSmiResultsPersister
 	TestProfilesPersister *BitCaskTestProfilesPersister
 }
 
+// Initialize will initialize the local provider
+func (l *DefaultLocalProvider) Initialize() {
+	l.ProviderName = "None"
+	l.ProviderDescription = []string{
+		"Ephemeral sessions",
+		"Environment setup not saved",
+		"No performance test result history",
+		"Free Use",
+	}
+	l.ProviderType = LocalProviderType
+	l.PackageVersion = "v0.0.1"
```

Comment:
Ideally, this is always the same as the Meshery Server version and is taken from the buildtime ldflags.

A future enhancement.

Reply:
Filed issue #2148 for follow up